src/encoded/tests/features/cart.feature

```diff
@@ -14,8 +14,7 @@ Feature: Cart
         Then I should see 3 elements with the css selector ".cart-toggle--in-cart"
         And I should see an element with the css selector ".cart__nav-button"
 
-        When I press "Data"
-        And I click the link with text that contains "High-throughput assays"
+        When I visit "/search/?type=FunctionalCharacterizationExperiment&type=FunctionalCharacterizationSeries&config=FunctionalCharacterization&datapoint=false&control_type!=*"
```

Comment:
By switching it to the link and not the navigation, are you skipping the testing of the navigation?  Come to think of it, now that there are two elements with the name "Experiment Search" will testing be a problem.

Reply:
yep, I am going around the fact that I can not uniquely press FCC Experiments vs regular experiments